src/components/Profile/ProfileContainer.js

```diff
@@ -3,29 +3,32 @@ import { withRouter } from 'react-router-dom';
 import { connect } from 'react-redux';
 import axios from 'axios';
 
-import { PROFILE_URL } from '../../utils/url-utils';
+import { MAIN_URL } from '../../utils/url-utils';
 import { DEFAULT_USER_ID } from '../../utils/constants';
 import { setUserProfile } from '../../redux/actions/actionCreators';
 
 import Profile from './Profile';
 
 class ProfileContainer extends Component {
   componentDidMount() {
+    const { setUserProfile } = this.props;
     let { userId } = this.props.match.params;
```

Comment:
Consider using destructuring here, it will fix the "Must use destructuring props assignment" linting error.

Reply:
/fxd